source/features/linkify-user-edit-history-popup.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'dom-chef';
+import {observe} from 'selector-observer';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function init(): void {
+	observe('details-dialog .Box-header .mr-3 img', {
+		add(element) {
+			const userName = (element as HTMLImageElement).alt.slice(1);
+			if (userName.includes('[bot]')) {
+				return;
+			}
```

Comment:
```suggestion
	observe<HTMLImageElement>('details-dialog .Box-header .mr-3 img:not([alt*="[bot]"])', {
		add(element) {
			const userName = element.alt.slice(1);
```

Reply:
The image generic type doesn't seem to work due to https://github.com/josh/selector-observer/issues/29